File: prueba2.c
#include <stdio.h>
#include <math.h>

int main(){
	int x1, x2, y1, y2;
	int z1, z2;
	printf("primer polinomio: da las dos constantes:");
	scanf("%i %i", &x1, &x2);
	printf("segundo polinomio: da las dos constantes:");
	scanf("%i %i", &y1, &y2);
	z1=x1+y1;
	z2=x2+y2;
	printf("  %ix+%i\n", x1, x2);
	printf("+ %ix+%i\n", y1, y2);
	printf("------\n");
	printf("+ %ix+%i\n", z1, z2);
	return 0;
}

File: prueba2.scala
object prueba {
  def main (args: Array[String]){
    println("Primer polinomio: da las dos constantes:")
    val x1=readInt()
    val x2=readInt()
    println("Segundo polinomio: da las dos constantes:")
    val y1=readInt()
    val y2=readInt()
    println("  "+x1+"x+"+x2)
    println("+ "+y1+"x+"+y2)
    println("------")
    val z1=x1+y1
    val z2=x2+y2
    println(" "+z1+"x+"+z2)
  }
}
